engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/savedGames/GameProvider.java

```diff
@@ -110,21 +111,42 @@ public static boolean isSavesFolderEmpty() {
     }
 
     /**
-     * Generates the game name based on the game number of the last saved game
+     * Generates the game name based on the game number of the last saved game Uses {@link
+     * GameProvider#DEFAULT_GAME_NAME_PREFIX} for resolve.
      */
     public static String getNextGameName() {
+        String nextGameName = getNextGameName(DEFAULT_GAME_NAME_PREFIX);
+        if (nextGameName.equals(DEFAULT_GAME_NAME_PREFIX)) {
+            return nextGameName + 1;
+        }
```

Comment:
Why is this necessary? shouldn't this be covered by `getNextGameName(gameName)`?

Reply:
Default name is "Game 1"
If move this it getNextGameName(gameName) then any custom gamenames will have suffix " 1"